packages/block-library/src/image/index.js

```diff
@@ -25,6 +25,13 @@ export const settings = {
 		'img', // "img" is not translated as it is intended to reflect the HTML <img> tag.
 		__( 'photo' ),
 	],
+	example: {
+		attributes: {
+			sizeSlug: 'large',
+			url: 'https://images.unsplash.com/photo-1549880339-d93e3072aef4',
+			caption: __( 'Snow covered mountain' ),
```

Comment:
Image(s) from Unsplash probably need a copyright note.

Reply:
We should bundle any image we use as well.

Reply:
Can't we use images hosted on wp.org instead? Bundling could be challenging as the public URL is not easily accessible here.